### PR TITLE
Daren

### PR DESCRIPTION
**Pull request by femee24**
Please check Thank You

### CODE REVIEW

File: modules/people/manifests/femee24.pp [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class people::femee24 {
-  require groups::osiris
-  notify { 'Hello ${::boxen_user}': }
+#  require groups::osiris
+notify { 'Hello ${::boxen_user}': }
+include people::impakt
```
Comment (devcsrj) [inline modules/people/manifests/femee24.pp]: You sure not to revert this back?

File: modules/people/manifests/impakt.pp [hunk 2]
```diff
@@ -0,0 +1,89 @@
+
+class people::impakt {
+
+  # Applications for Daren Klamer Only
+
+  $DL_SQUIRREL    = 'http://nchc.dl.sourceforge.net/project/squirrel-sql/3-snapshots/snapshot-20150623_2101/squirrel-sql-snapshot-20150623_2101-MACOSX-install.jar'
+  $DL_TALEND      = 'http://talend.dreamhosters.com/tos/release/V6.0.0/TOS_DI-20150702_1326-V6.0.0.zip'
```
Comment (devcsrj) [inline modules/people/manifests/impakt.pp]: Remove these 2 lines 6, 7

File: modules/people/manifests/impakt.pp [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+  include diffmerge
+  include solr
+  include sublime_text
+  include textwrangler
+  include virtualbox
```
Comment (devcsrj) [inline modules/people/manifests/impakt.pp]: I don't think he'll need `diffmerge`, `solr`, `textwrangler`

File: modules/people/manifests/impakt.pp [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  }
+
+
+    package { 'dbvisualizer':
+      ensure          => installed,
+      provider        => 'brewcask',
+      install_options => []
+    }
+
+    package { 'pgadmin3':
```
Comment (devcsrj) [inline modules/people/manifests/impakt.pp]: Remove `pgadmin3`

File: modules/people/manifests/impakt.pp [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  }
+
+  package { 'remote-desktop-manager':
+    ensure          => installed,
+    provider        => 'brewcask',
+    install_options => ['--no-binaries','--appdir=/Applications'],
+  }
+
+
+    package { 'dbvisualizer':
```
Comment (devcsrj) [inline modules/people/manifests/impakt.pp]: Remove this one too

File: modules/people/manifests/impakt.pp [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+      install_options => []
+    }
+
+    package { 'pgadmin3':
+      ensure          => installed,
+      provider        => 'brewcask',
+      install_options => []
+    }
+
+    package { 'mysqlworkbench':
```
Comment (devcsrj) [inline modules/people/manifests/impakt.pp]: Remove `mysqlworkbench`

File: modules/people/manifests/impakt.pp [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+      require  => Package['java']
+    }
+
+    package { 'tomcat':
+      ensure   => 'latest',
+      provider => 'homebrew',
+      require  => Package['java']
+    }
+
+    package { 'jd-gui':
```
Comment (devcsrj) [inline modules/people/manifests/impakt.pp]: Remove this one too

File: modules/people/manifests/femee24.pp [hunk 8]
```diff
@@ -1,5 +1,5 @@
 class people::femee24 {
   require groups::osiris
-  notify { 'Hello ${::boxen_user}': }
+notify { 'Hello ${::boxen_user}': }
```
Comment (devcsrj) [inline modules/people/manifests/femee24.pp]: Revert indention

File: modules/people/manifests/impakt.pp [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+    provider        => 'brewcask',
+    install_options => ['--no-binaries','--appdir=/Applications'],
+  }
+
+  package { 'java':
+    ensure          => installed,
+    provider        => 'brewcask',
+  }
+
+  package { 'filezilla':
```
Comment (devcsrj) [inline modules/people/manifests/impakt.pp]: I thought we're using `cyberduck` instead? Or was this requested by him?

**Comment by devcsrj** [comment]
@pongstr This LGTM. I'll merge it now.

**Comment by pongstr** [comment]
:+1:

**Pull request merged**